gtsam/hybrid/HybridNonlinearFactor.h

```diff
@@ -90,25 +90,27 @@ class HybridNonlinearFactor : public HybridFactor {
    * Will be typecast to NonlinearFactor shared pointers.
    * @param keys Vector of keys for continuous factors.
    * @param discreteKeys Vector of discrete keys.
-   * @param factors Vector of nonlinear factors.
+   * @param factors Vector of nonlinear factor and scalar pairs.
    * @param normalized Flag indicating if the factor error is already
    * normalized.
    */
   template <typename FACTOR>
-  HybridNonlinearFactor(const KeyVector& keys, const DiscreteKeys& discreteKeys,
-                        const std::vector<std::shared_ptr<FACTOR>>& factors,
-                        bool normalized = false)
+  HybridNonlinearFactor(
+      const KeyVector& keys, const DiscreteKeys& discreteKeys,
+      const std::vector<std::pair<std::shared_ptr<FACTOR>, double>>& factors,
```

Comment:
Same: don’t have a vector constructor

Reply:
Updated/Removed all vector constructors in the Hybrid API. :)
HybridNonlinearFactor and HybridGaussianFactor only accept a single discrete key, while HybridGaussianConditional doesn't have a vector constructor at all.